configgtm-v1/service.go

```diff
@@ -0,0 +1,60 @@
+package configgtm
+
+import (
+	"github.com/akamai/AkamaiOPEN-edgegrid-golang/edgegrid"
+        "net/http"
+        "net/http/httputil"
+        "fmt"
+
+        log "github.com/sirupsen/logrus"
+)
+
+var (
+	// Config contains the Akamai OPEN Edgegrid API credentials
+	// for automatic signing of requests
+	Config edgegrid.Config
+
+        debug bool
+)
+
+// Init sets the GTM edgegrid Config
+func Init(config edgegrid.Config) {
+	Config = config
+
+        if config.Debug {
+                log.SetLevel(log.DebugLevel)
+        }
+
+        // used as trouble shooting hook til logging implemented.
+        debug = false
+ 
+        log.Debugf("Log debug level set to  %v", config.Debug)
+
+}
+
+// Utility func to print http req
+func printHttpRequest(req *http.Request, body bool) {
+
+        if !debug {
```

Comment:
I would look at using the `sirupsen/logrus` package and the `AKAMAI_LOG` env variable to control output (see [here for how we do it in Akamai CLI](https://github.com/akamai/cli/blob/master/log.go) — I intend to migrate this to the common package at some point for all Golang CLI packages to use also)

Reply:
refactored to use logrus. Added log.go to edgegrid package. Made SetupLogging generic for anyone to use.